### PR TITLE
More professional-looking presentation of the bibliography

### PR DESCRIPTION
**Pull request by chrishtr**
1. Names first
2. Italicized names
3. Year published
4. Publisher, if any

I verified online that this is close enough to "correct" according to rules of style.

I chose not to write the names in "last name, first name" format, since our book has a more informal style than most.

### CODE REVIEW

File: book/bibliography.md [hunk 1]
```diff
@@ -2,71 +2,36 @@
...
-[A Brief History of JavaScript](https://auth0.com/blog/a-brief-history-of-javascript/)
+Sebastian Peyrott. [*A Brief History of JavaScript*](https://auth0.com/blog/a-brief-history-of-javascript/), 2017.
 
-: Sebastian Peyrott
+Alan Grosskurth and Michael W. Godfrey. [*A Reference Architecture for Web Browsers*](https://grosskurth.ca/papers/browser-refarch.pdf), 2005.
 
-[A Reference Architecture for Web Browsers](https://grosskurth.ca/papers/browser-refarch.pdf)
+Jesse James Garrett. [*Ajax: A New Approach to Web Applications*](https://www.semanticscholar.org/paper/Ajax%3A-A-New-Approach-to-Web-Applications-Garrett/c440ae765ff19ddd3deda24a92ac39cef9570f1e), 2005.
+ 
+James D. Foley, Andries van Dam, Steven K. Feiner, John Hughes, Morgan McGuire, David F. Sklar, and Kurt Akeley. [*Computer Graphics: Principles and Practice*](https://en.wikipedia.org/wiki/Computer_Graphics:_Principles_and_Practice). Addison-Wesley, 1995.
```
Comment (pavpanchekha) [inline book/bibliography.md:11]: Maybe no link on this one, since the actual link is kind of bullshit.

File: book/bibliography.md [hunk 2]
```diff
@@ -2,71 +2,36 @@
...
+Mariko Kosaka. [*Inside Look at Modern Web Browsers*](https://developer.chrome.com/blog/inside-browser-part1), 2018.
 
-[*High-performance Browser Networking*](https://hpbn.co/)
+Matt Brubeck. [*Let's Build a Browser Engine*](https://limpet.net/mbrubeck/2014/08/08/toy-layout-engine-1.html), 2014.
 
-: Ilya Grigorik
+Chris Harrelson. [*RenderingNG: Ready for the next generation of web content*](https://developer.chrome.com/docs/chromium/renderingng), 2021.
 
-[How Browsers Work](https://taligarsiel.com/Projects/howbrowserswork1.htm)
+Tomas Akenine-Möller, Eric Haines, Naty Hoffman, Angelo Pesce, Michał Iwanicki, and Sébastien Hillaire. [*Real-Time Rendering*](https://www.realtimerendering.com/). CRC Press, 2018.
```
Comment (pavpanchekha) [inline book/bibliography.md:31]: We’ll need to check later proofs to make sure the Unicode names come through.

**Comment by pavpanchekha** [comment]
Do we want to alphabetize? Did you already? The order seems pretty random.

**Comment by chrishtr** [comment]
> Do we want to alphabetize? Did you already? The order seems pretty random.

Now sorted by last name, per official guidance.

**Review by pavpanchekha — APPROVED** [review]

**Pull request merged**